src/components/AdjMatrix/AdjMatrixMethods.ts

```diff
@@ -49,6 +49,7 @@ export class View {
   private highlightedNodes: { [key: string]: any[] } = {};
   private columnSelectedNodes: any[] = [];
   private mouseOverEvents: any;
+  private maxVal: number = 0
```

Comment:
```suggestion
  private maxVal: number = -Infinity;
```

Even though `this.matrix` contains only `0` and `1`, setting the initial `maxVal` this way ensures that the max value computation below can never fail.